packages/trader/src/AppV2/Components/BottomNav/bottom-nav.scss

```diff
@@ -33,5 +36,10 @@
                 fill: var(--core-color-opacity-coral-600);
             }
         }
+
+        &-label {
+            font-size: var(--core-fontSize-50);
```

Comment:
In current design bottom nav font style is different from production, because in Figma designers used Navigation - bottom bar component which haven't been provided by Quill yet.